### PR TITLE
Update home page

### PR DESCRIPTION
**Pull request by shinedesigns**
This addresses all requested home page updates, including the Covid-19 alert banner

### CODE REVIEW

File: _data/partners.yml [hunk 1]
```diff
@@ -200,4 +200,31 @@ partners:
...
+  - title: "WNCAP"
+    image: "partner-wncap.png"
+    link: "https://wncap.org/"
+    include-page: ["home-partners", "strategic-partners"]
+  - title: "12 Baskets"
+    image: "partner-12-baskets.png"
+    link: "https://www.ashevillepovertyinitiative.org/"
+    include-page: ["home-partners", "strategic-partners"]
+  - title: "Julian F. Keith Alcohol and Drug Abuse Treatment Center"
+    image:
```
Comment (j00bar) [inline _data/partners.yml]: We definitely need an image asset here, I think.

Comment (j00bar) [reply _data/partners.yml]: Oh, I see - you altered the section template for when we lack an image asset?

Comment (shinedesigns) [reply _data/partners.yml]: Correct -- manifest said there was no image, and to just link the text.

**Comment by j00bar** [comment]
So, Jen doesn't really get to preview this stuff without me compiling it for her and showing her... So...

1. The color change looks awful. Let's revert that.
2. The 1st sentence of the elevator pitch needs to be more visually prominent - either in size, italics, or something.
3. Jen doesn't like how dense the shift of the "bubbles" to the bullet points in the `about-seekhealing.md` stuff is. She will be in touch with a mock up to guide some different design.
4. The QZ article section just isn't great. The headline text is awkward - maybe "In the news" or something. And the actual URL shouldn't be visible, it should just be clickable.
5. The image assets for the partners need some handling. It's awkward of course that we have some banner-style assets for a square placement. But for those, can we normalize the asset sizes some so that they look less disjoint? See the WNCAP logo mods I did as an example of that.

Thanks, brother.

**Comment by howtohumanbetter** [comment]
hey! Yes, that's right. Thanks for summarizing this @j00bar and for the awesome start @shinedesigns 

For #3: I guess the purple blocks are fine, unless someone can imagine a better design. I'd just like to change the words that are inside them.

**Comment by shinedesigns** [comment]
To be clear, this PR was intended to address (urgent) content updates and not presentation layer.

Jennifer and I had discussed working on aesthetics in a separate branch, essentially. I'm trying to be mindful of your time; but if you would rather I asserted more artistic liberty as I go, I will take more time and ownership of design and not push anything less than beautiful to the master.

> So, Jen doesn't really get to preview this stuff without me compiling it for her and showing her... 

I'll throw my fork up somewhere and text you both the URL so we can shorten this feedback loop.

> 1. The color change looks awful. Let's revert that.

I actually think that magenta was awful and that you were right to reject it; and if a blue had not been specifically requested, I would have used the existing light purple. Single colors are strong, let's try that first.

> 2. The 1st sentence of the elevator pitch needs to be more visually prominent - either in size, italics, or something.

Ok, sure.

> 3. Jen doesn't like how dense the shift of the "bubbles" to the bullet points in the `about-seekhealing.md` stuff is. She will be in touch with a mock up to guide some different design.

I got you.

> 4. The QZ article section just isn't great. The headline text is awkward - maybe "In the news" or something. And the actual URL shouldn't be visible, it should just be clickable.

This was simply copied "verbatim" from /press, so I'll take care of both sections. I'm sure we can change the links to buttons, or something.

> 5. The image assets for the partners need some handling. It's awkward of course that we have some banner-style assets for a square placement. But for those, can we normalize the asset sizes some so that they look less disjoint? See the WNCAP logo mods I did as an example of that.

Flexbox. 👍

**Comment by howtohumanbetter** [comment]
cool thank you @shinedesigns . I trust you on the color choice. For the bubbles let's just go with the bulletted list for now then and I'd love to revisit a better aesthetic in the separate branch you described. Thanks for taking care with the urgency. @j00bar if @shinedesigns makes these little changes can we commit later today?

**Comment by shinedesigns** [comment]
Still working on these but wanted to add progress thus far.

**Comment by shinedesigns** [comment]
This should be all set to merge... Have a look at my fork and let me know what you think.

**Pull request merged**